test/components/structures/MessagePanel-test.js

```diff
@@ -191,50 +263,4 @@ describe('MessagePanel', function() {
             }, 100);
         }, 100);
     });
-
-    it('shows only one ghost when the RM moves twice', function() {
```

Comment:
I should say: this test was really testing that the animations didn't break (https://github.com/matrix-org/matrix-react-sdk/pull/254). The animations will now work fine however many ghosts we end up showing so this test is no longer useful as it shows multiple ghosts by design.